plutus-core/plutus-core.cabal

```diff
@@ -5,8 +5,8 @@ license: Apache-2.0
 license-files:
   LICENSE
   NOTICE
-maintainer: vanessa.mchale@iohk.io
-author: Vanessa McHale
+maintainer: michael.peyton-jones@iohk.io
+author: Plutus Core Team
```

Comment:
:clap:

Reply:
We still need a better email for this...